### PR TITLE
Three small fixes

### PR DESCRIPTION
**Pull request by NikolausDemmel**
I can also split them up in 3 PRs if needed. For description see the commit messages.

### CODE REVIEW

File: src/software/ui/SfM/adjacency_matrix_viewer/mainframe.cpp [hunk 1]
```diff
@@ -63,15 +63,15 @@ MainFrame::MainFrame(const QString &name, const Document & doc, QWidget *parent)
   QHBoxLayout *label_layout = new QHBoxLayout;
   QLabel *label = new QLabel(name);
   QLabel *label2 = new QLabel(tr("Pointer Mode"));
-  QToolButton *select_mode_button = new QToolButton;
```
Comment (pmoulon) [inline src/software/ui/SfM/adjacency_matrix_viewer/mainframe.cpp]: 👍 
My mistake was to do not post-fix the mainframe class variable with a `_`.

File: src/software/SfMWebGLViewer/common/main.js [hunk 2]
```diff
@@ -245,7 +245,7 @@ function onMouseMove( e )
       var new_rad = d * Math.sin( arad * 0.5 ); 
       trackball.setRadius( new_rad * 0.8 ); 
     }
-    else if (e.altKey )
+    else if (e.altKey || e.ctrlKey)
```
Comment (pmoulon) [inline src/software/SfMWebGLViewer/common/main.js]: Which is your motivation behind adding this key?
Does another software you are using can use ctrl to pan?

File: src/software/SfM/tutorial_demo.py.in [hunk 3]
```diff
@@ -19,10 +19,10 @@ import subprocess
 import sys
 
 def get_parent_dir(directory):
-    import os
```
Comment (pmoulon) [inline src/software/SfM/tutorial_demo.py.in]: thx

File: src/software/SfMWebGLViewer/common/main.js (hunk 2 above)
Comment (NikolausDemmel) [reply src/software/SfMWebGLViewer/common/main.js]: See commit message:

> On Linux systems like Ubuntu with Gnome or Unity, alt+drag is bound to
move the window, so this alternative allows panning also in that
situation.

**Comment by pmoulon** [comment]
Sorry for did not answer sooner.
I will take a look to the PR during this week end.

**Comment by pmoulon** [comment]
@NikolausDemmel Thank you!
Code merged.
Looking forward your future PRs ;-)

**Comment by NikolausDemmel** [comment]
Thanks Pierre!

> Looking forward your future PRs ;-)

Well, hopefully :-). We'll see.

**Pull request merged**